actionpack/lib/action_dispatch/routing/mapper.rb

```diff
@@ -48,10 +48,14 @@ def serve(req)
           @strategy.call @app, req
         end
 
-        private
-          def constraint_args(constraint, request)
-            constraint.arity == 1 ? [request] : [request.path_parameters, request]
+        def constraint_args(constraint, request)
+          if constraint.respond_to?(:arity) && constraint.arity == 1
+            [request]
+          else
+            [request.path_parameters, request]
           end
+        end
+        private :constraint_args
```

Comment:
Why did you changed the way the private method is defined?

Reply:
@rafaelfranca I changed it because it was like that in other places of the same file 😊

Reply:
Lines 187 and 192 for instance.

Reply:
Better to leave as it is. Line 255 is using the previous code.